PreProc/inputdata/atm/cam/ggas/create_TIPMIP_emission_file.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import sys
+import xarray as xr
+from scipy.io import netcdf
+import pandas as pd
+
+
+# load grid area from fx
+dataset = xr.open_dataset('/mnt/bgcdata-ns2980k/ffr043/TipESM/data/NorESM2-LM/1pctCO2/fx/areacella_fx_NorESM2-LM_1pctCO2_r1i1p1f1_gn.nc')
+areavar = dataset['areacella']
+totarea = areavar.sum().values
+
+# load example emission file
+inpath = '/mnt/bgcdata-ns2980k/ffr043/TipESM/data'
+dataset = xr.open_mfdataset(inpath+'/emissions-cmip6_CO2_anthro_surface_175001-201512_fv_1.9x2.5_c20181011.nc', decode_times=False, format="NETCDF3_64BIT")
+co2var = dataset['CO2_flux']
+
+# distribute emissions over time and space
+# translated to python from matlab based on script by J. Schwinger
+nyears = 250
+nmonth = nyears * 12
+nlon = 144
+nlat = 96
+start_year = 1
+time_bnds = np.zeros((2, nmonth))
+time = np.zeros(nmonth)
+date = np.zeros(nmonth, dtype=np.int32)
+co2flx = np.zeros((nmonth, nlat, nlon))
+dayim = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
+days = np.cumsum([0] + dayim[:-1]) + 1
+daye = np.cumsum(dayim)
+
+# calculate emissions based on TCRE (Arora et al., 2020)
+TCRE = 1.32 #°C EgC-1
+E = 1000 * 0.02 / TCRE #GtC yr-1
+Edata =  np.ones((nmonth, 1)) * E
```

Comment:
`Edata` is the amount of emissions in PgC yr-1 that will give 0.02 degree warming, right? 

Then I believe there is an error in line 53, because there you divide by `dt`, which is the number of seconds in a month, but it should be nb of seconds per year.

Reply:
Ok, I see, but this leads actually to different emission rates in different month (because dt is calculated based on the number of days per month). Although the cumulative emission over a year will be correct, you might want to avoid this. So I would suggest to use Edata as calculated in line 36 and then divide by seconds_per_year in line 53 instead of `/dt/12`

Reply:
Resolved by latest commit.